app/api/events.py

```diff
@@ -552,6 +559,8 @@ def before_update_object(self, event, data, view_kwargs):
             start_image_resizing_tasks(event, data['original_image_url'])
 
     def after_update_object(self, event, data, view_kwargs):
+        if EventDetail.event_name != event.name:
+            rename_rocketchat_room(event=event)
```

Comment:
This should be a celery task so that it doesn't slow down the event PATCHH request